sqlglot/dialects/oracle.py

```diff
@@ -328,3 +364,22 @@ def queryoption_sql(self, expression: exp.QueryOption) -> str:
         def coalesce_sql(self, expression: exp.Coalesce) -> str:
             func_name = "NVL" if expression.args.get("is_nvl") else "COALESCE"
             return rename_func(func_name)(self, expression)
+
+        def multitable_inserts_sql(self, expression: exp.MultitableInserts) -> str:
+            kind = self.sql(expression, "kind").upper()
+            inserts = []
+            for conditional_insert in t.cast(t.List[exp.ConditionalInsert], expression.expressions):
+                buffer = ""
+                if conditional_insert.expression:
+                    buffer += f"WHEN {self.sql(conditional_insert.expression)} THEN "
+                if conditional_insert.args["else_"]:
+                    buffer += "ELSE "
+                insert = t.cast(exp.Insert, conditional_insert.this)
+                insert_sql = self.sql(insert)
+                if insert_sql.startswith("INSERT "):
+                    insert_sql = insert_sql[len("INSERT ") :]
+                buffer += insert_sql
```

Comment:
This isn't right, you need to also add a method to generate `ConditionalInsert` and then use `self.expressions` or something.